custom_components/apsystemsapi_local/sensor.py

```diff
@@ -138,16 +137,11 @@ def device_info(self) -> DeviceInfo:
             model="EZ1-M",
         )
 
-    @property
-    def extra_state_attributes(self):
-        """Return entity specific state attributes."""
-        return self._attributes
-
     async def async_update_data(self):
         try:
             data = await self._api.get_output_data()
             self.update_state(data)
-            self._attr_available = True
+            self._attr_available = False
```

Comment:
This "_attr_available" doesn't have anything to do with attributes. It just tells HA if the sensor is available or greyed out.

Reply:
Thx, didn't know. It's fixed now.